DBSchema/users_address.sql

```diff
@@ -0,0 +1,53 @@
+-- MySQL dump 10.13  Distrib 8.0.38, for Win64 (x86_64)
+--
+-- Host: localhost    Database: users
+-- ------------------------------------------------------
+-- Server version	8.0.39
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!50503 SET NAMES utf8 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `address`
+--
+
+DROP TABLE IF EXISTS `address`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `address` (
+  `Address_ID` int NOT NULL AUTO_INCREMENT,
+  `USER_ID` int DEFAULT NULL,
+  `Address_Type` enum('Primary','Office') DEFAULT NULL,
+  `Building_Name` varchar(100) DEFAULT NULL,
+  `Building_No` int DEFAULT NULL,
+  `Street` varchar(100) DEFAULT NULL,
+  `City` varchar(50) DEFAULT NULL,
+  `State` varchar(50) NOT NULL,
+  `Pincode` int NOT NULL,
+  `Country` varchar(50) NOT NULL,
+  `Hobbies` set('Reading','Traveling','Cooking','Hiking','Music','Sports') DEFAULT NULL,
```

Comment:
Is this  a fixed set of values for now? Will we be treating this as a dropdown in the future?

Reply:
Yes, this is the fixed set of values. We can add more values in db schemas if we want to showcase this in dropdown

Reply:
I think we should keep it a free-form text, we should code to auto-suggest based on values in DB. Maybe you can sync on this with Alind once. Also, I think he meant this field to a be a multi-select, meaning we should be able to have more than 1 value. Do we need to make any db change for that?

Reply:
I already had discussion with Alind regarding the same. Below are some challenges to store hobbies with free-form text.

1) redundancy of data 
Ex. If 2 users enters singing 2 times, will you take it as 2 hobbies? and then again you maintain mapping of users and hobbies, that will be tedious task.

2) We can store comma separated values as a string for user but in that case all coding tasks like manipulation of the strings and storing it into db will be our headache.

3) We can introduce joins in our table but it also a complex operations for only hobbies thing

4) (Our solution) After All above discussion, we have decided to use set for simple storing.

Reply:
Ok then we should be good for now. We will look into this later.